scripts/get_latest_release.py

```diff
@@ -14,19 +12,11 @@
    # Get all the releases (include pre-releases) and pick the first of them
```

Comment:
Why get releases anymore? Is this for rc or are there other versions we are marking as not release versions ?